pages/reviews.tsx

```diff
@@ -0,0 +1,38 @@
+import * as React from 'react'
+import type { NextPage } from 'next'
+import Container from '@mui/material/Container'
+import Typography from '@mui/material/Typography'
+import Box from '@mui/material/Box'
+import Button from '@mui/material/Button'
+import Link from '../src/Link'
+import ProTip from '../src/ProTip'
+import Copyright from '../src/Copyright'
+
+const About: NextPage = () => {
```

Comment:
Should `About` be `Reviews` here?

Reply:
Probably, but I"m not sure because I just duplicated the "about" page and renamed the file "reviews." 
I might just delete the reviews stuff from this PR.